flask/app/auth.py

```diff
@@ -1,14 +1,5 @@
-from flask import (
-    Blueprint,
-    flash,
-    redirect,
-    render_template,
-    request,
-    session,
-    url_for,
-)
+from flask import ( Blueprint, flash, redirect, render_template, request, session, url_for)
```

Comment:
Flake8 will complain about this too because the line is too long (limit 79 chars acc. to PEP8)

In general try not to change files that have nothing to do with the issue/PR

Reply:
fixed